powernet-project/powernet
# Drop swagger

# Code changes
1. removed everything associated with swagger-rest-framework
2. fixed some js file names and changed import lines in html files

# Validation suggestions
1. run locally - check if localhost:8000/api/v1/ is returning Page not found (404) - if so that means swagger has successfully been removed.
2. make sure to sign in with farm account and check localhost:8000/energy_summary and navigate through to see if everything is working correctly
